project/Dependencies.scala

```diff
@@ -17,7 +17,7 @@ object Dependencies {
   val commontTextV = "1.12.0"
   val googleDiffV = "1.3.0"
   val jgitV = "5.13.3.202401111512-r"
-  val metaconfigV = "0.13.0"
+  val metaconfigV = "0.14.0"
```

Comment:
bumping this at the same time as doc to prevent
```
[error] 	* org.scalameta:metaconfig-typesafe-config_2.13:0.14.0 (early-semver) is selected over {0.13.0}
[error] 	    +- org.scalameta:mdoc_2.13:2.6.2                      (depends on 0.14.0)
[error] 	    +- org.scalameta:mdoc-cli_2.13:2.6.2                  (depends on 0.14.0)
[error] 	    +- org.scalameta:metaconfig-docs_2.13:0.13.0          (depends on 0.13.0)
[error] 	    +- ch.epfl.scala:scalafix-core_2.13:0.0.0+1-804f3a2e-SNAPSHOT (depends on 0.13.0)
```